prow/jobs/control-plane/components/provisioner/provisioner-integration-test.yaml

```diff
@@ -0,0 +1,38 @@
+presubmits: # runs on PRs
+  kyma-project/control-plane:
+    - name: pre-main-provisioner-e2e-tests
+      annotations:
+        description: "It tests the component's correctness"
+        owner: framefrog
+        pipeline.trigger: "pr-submit"
+        pipeline.type: "integration"
```

Comment:
@kyma-project/prow do you happen to know if that line is required? From what I see all jobs are using the same type.